{{cookiecutter.project_shortname}}/package.json

```
@@ -29,30 +29,29 @@
     "ramda": "^0.26.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.5.4",
-    "@babel/preset-env": "^7.5.4",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/core": "^7.22.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
+    "@babel/preset-env": "^7.22.2",
+    "@babel/preset-react": "^7.22.3",
+    "@plotly/dash-component-plugins": "^1.2.3",
     "@plotly/webpack-dash-dynamic-import": "^1.2.0",
-    "@plotly/dash-component-plugins": "^1.2.0",
-    "babel-eslint": "^10.0.2",
-    "babel-loader": "^8.0.6",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^9.1.2",
     "copyfiles": "^2.1.1",
-    "css-loader": "^3.0.0",
+    "css-loader": "^6.8.1",
     "eslint": "^6.0.1",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-import": "^2.18.0",
     "eslint-plugin-react": "^7.14.2",
-    "prop-types": "^15.7.2",
+    "prop-types": "^15.8.1",
     "react": "^16.8.6",
-    "react-docgen": "^4.1.1",
+    "react-docgen": "^5.4.3",
     "react-dom": "^16.8.6",
-    "styled-jsx": "^3.2.1",
     "style-loader": "^0.23.1",
```

```suggestion
    "style-loader": "^3.3.3",
```
What's the pattern for which packages you updated and which you didn't? Let's try and get a few more of these updated while we're at it. I think if we use the latest `style-loader` and `css-loader` we just need to change the webpack config for css to:
```
use: ['style-loader', 'css-loader']
```